### DISCUSSION TITLE
When will be the new version 21.x get released?

### DISCUSSION THREAD

**Discussion by dev-venkatesh** [OpenMage/magento-lts#4354] [Questions & Answers]

**Comment by dev-venkatesh** [comment]
Can you please put comments on this?

**Comment by sreichel** [comment]
I dont know. Latest changes made to main branch (v20) are not in v21.

@colinmollenhour any ideas?

**Reply by colinmollenhour** [reply]
I'm not well informed on what all of the BC changes are, but v21 should basically become the current state of next, with all changes from main included. There are a ton of "Merge 'main'" commits though, so I hate to merge all of them... May need to do a rebase and force push to clean it up.

I'm onboard with the release, just let me know if there are any parts you need help with.

**Comment by dev-venkatesh** [comment]
I don't want to switch to Magento-2. How long the **magento-lts** will be supported.?
PHP is continuously releasing new versions/patch. Either it it long support or quit for short time.?

@sreichel Can you please reply for this?

**Reply by sreichel** [reply]
Hey @dev-venkatesh,

as you can see on nearly daily progress OpenMage is good supported. No nned to think about M2.

There will be a new **v20** release this year - just waiting for https://github.com/OpenMage/magento-lts/discussions/4275#discussioncomment-11354832.

For v21 release ... i dont really care about this branch. Sadly. There a good features in, but also changes that could have been added in another way. There are some PRs that could be backported, ... later ... or with help.

Btw ... You can speed it up with testings. Any help, any review is welcome.
